cmd/kubeless/ingress.go

```diff
@@ -18,9 +18,9 @@ import (
 )
 
 var ingressCmd = &cobra.Command{
```

Comment:
if this file will be shared whenever a different `route` type exists this should be `routeCmd`